.mergify.yml

```diff
@@ -1,10 +1,49 @@
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
 
 pull_request_rules:
-  - name: automatically merge scala-steward's PRs
-    conditions:
-      - author=scala-steward
-      - body~=labels:.*semver-patch.*
-      - status-success=build
-    actions:
-      merge:
-        method: merge
+- name: merge scala-steward's PRs
+  conditions:
+  - author=scala-steward
+  - or:
+    - body~=labels:.*early-semver-patch
+    - body~=labels:.*early-semver-minor
+  - status-success=Build and Test (ubuntu-latest, 2.12.15, temurin@17)
```

Comment:
This will merge any binary-compatible change, according to semver. e.g.
✅ 1.2.3 -> 1.2.4
✅ 1.2.3 -> 1.3.0
✅ 0.4.2 -> 0.4.3
❌ 0.4.2 -> 0.5.0
❌ 1.2.3 -> 2.0.0

Reply:
Cool, that was the previous logic I think

Reply:
The previous logic was:
```yaml
- body~=labels:.*semver-patch.*
```
Which would not merge `1.2.3 -> 1.3.0` I think.

Reply:
Ok, that'd be good now that we are updating the minor very often

Reply:
Yup. Although it will not merge a minor `0.4.2` -> `0.5.0` because that may not be binary compatible.